doc/Overview/Logging/LoggingOverview.md

```diff
@@ -1,108 +1,174 @@
 
-  - **Trace** : Used for parts of a method to capture a flow.
-  - **Debug** : Used for diagnostics information.
-  - **Information** : Used for general successful information. Generally the default minimum.
-  - **Warning** : Used for anything that can potentially cause application oddities. Automatically recoverable.
-  - **Error** : Used for anything that is fatal to the current operation but not to the whole process. Potentially recoverable.
-  - **Critical** : Used for anything that is forcing a shutdown to prevent data loss or corruption. Not recoverable.
+Use the following convention as a guide when logging messages:
 
-```csharp
-private IHost Host { get; }
+| Level | Description |
+|-------|-------------|
+| Trace | Used for parts of a method to capture a flow. |
+| Debug | Used for diagnostics information. |
+| Information | Used for general successful information. Generally the default minimum. |
+| Warning | Used for anything that can potentially cause application oddities. Automatically recoverable. |
+| Error | Used for anything that is fatal to the current operation but not to the whole process. Potentially recoverable. |
+| Critical | Used for anything that is forcing a shutdown to prevent data loss or corruption. Not recoverable. |
+
+Because these extension methods are scoped to a specific log level, the standard `Log()` method on the `ILogger<T>` interface does not need to be used.
+
+## Configuring Logging Output
+
+The `UseLogging()` extension method accepts an optional `Action<ILoggingBuilder>` parameter that can be used to configure the logging output.
 
-public App()
+### Setting the Minimum Log Level
+
+The minimum log level can be set by calling the `SetMinimumLevel()` extension method on the `ILoggingBuilder` instance. If no LogLevel is specified, logging defaults to the `Information` level which may not always be desirable. The following example shows how to change the minimum log level to `Debug`:
+
+```csharp
+protected override void OnLaunched(LaunchActivatedEventArgs e)
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseLogging()
-        .ConfigureLogging(logBuilder =>
-                {
-                    logBuilder
-                        .SetMinimumLevel(LogLevel.Debug)
-                        .XamlLogLevel(LogLevel.Information)
-                        .XamlLayoutLogLevel(LogLevel.Information);
-                })
-        .Build();
-    // ........ //
-}
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host    
+#if DEBUG
+            .UseEnvironment(Environments.Development)
+#endif
+            .UseLogging(configure:
+                (context, services) =>
+                    services.SetMinimumLevel(
+                    context.HostingEnvironment.IsDevelopment() ?
+                        LogLevel.Trace :
+                        LogLevel.Error)
+                    )
+        });
+...
```

Comment:
Can you provide more of an example here of what log message would/wouldn't be emitted based on this configuration